nautobot/core/graphql/utils.py

```diff
@@ -1,6 +1,68 @@
+import graphene
+
+
+from django_filters.filters import BooleanFilter, NumberFilter, MultipleChoiceFilter
+from nautobot.utilities.filters import MultiValueNumberFilter
+
+
 def str_to_var_name(verbose_name):
     """Convert a string to a variable compatible name.
     Examples:
         IP Addresses > ip_addresses
     """
     return verbose_name.lower().replace(" ", "_").replace("-", "_")
+
+
+def get_filtering_args_from_filterset(filterset):
```

Comment:
Mind changing the `filterset` variable to `filterset_class` throughout this function for clarity and consistency with the graphene_django function?